Makefile

```diff
@@ -32,3 +32,8 @@ unittests: phpdev
 
 # Perform all tests that don't require an install.
 check: checkstatic unittests
+
+# Configure githooks for this repository to be .githooks instead of the default
+# .git/hooks/ (which is ignored).
```

Comment:
This belongs in the documentation, not the Makefile

Reply:
I changed it to "Configures..." to make it more clear that the comment is describing what the command does, not telling the user to do an additional action.

Reply:
But the user should be told to do an additional action in the documentation if they so desire.. we shouldn't be changing their git configuration in the Makefile

Reply:
@johnsaigle documentation for this could be added in the developer section of the new wiki

Reply:
I added a note to CONTRIBUTING since we don't have a document for this kind of thing on the new Wiki yet

Reply:
The CONTRIBUTING shouldn't say "run make githooks". "githooks" isn't a reasonable make target. The CONTRIBUTING.md should say "run git config core.hooksPath .githooks or copy .githooks/pre-push to your .git/hooks directory"

Reply:
@driusan why ? because it's a one time configuration ?

what makes a reasonable make target ?

Reply:
because it's configuring the user's software (and doing it in an opinionated way), not building the software. Make is a build tool.